include/tudat/astro/ground_stations/transmittingFrequencies.h

```diff
@@ -200,20 +201,23 @@ class PiecewiseLinearFrequencyInterpolator: public StationFrequencyInterpolator
             // If there are discontinuities
             if ( startTimes_.at( i ) != endTimes_.at( i - 1 ) )
             {
-                // If the start and end times are inconsistent throw error
-                if ( endTimes_.at( i - 1 ) > startTimes_.at( i ) )
-                {
-                    throw std::runtime_error(
-                            "Error when creating piecewise linear frequency interpolator: inconsistency between ramp end "
-                            "time (" + std::to_string( double( endTimes_.at( i - 1 ) ) ) + ") and start time of the following ramp (" +
-                            std::to_string( double( startTimes_.at( i ) ) ) + "); the end is smaller than the start time." );
-                }
-                // If there are gaps in the data save that information
-                else
-                {
-                    invalidTimeBlocksStartTimes_.push_back( endTimes_.at( i - 1 ) );
-                    invalidTimeBlocksEndTimes_.push_back( startTimes_.at( i ) );
-                }
+//                // If the start and end times are inconsistent throw error
```

Comment:
We should think about what to do with this, it's for the case where the ramp tables overlap. I commented it out to keep it from crashing, but we should think of a better solution for it:
i) choosing one of the tables, and resetting the time range of the other
ii) giving a warning
iii) giving an exception
Also, we could check if the two ramp tables are really different, or have the same frequencies (possiby referenced at a different date)

Reply:
I can try to address before the merge, what do you think?

Reply:
After our conversation, I'll open an issue once we merge